### PR TITLE
[Common] Lecture QuerySetting

### PR DESCRIPTION
**Pull request by TaerangLee**

### CODE REVIEW

File: shared/api/common/src/hooks/lecture/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export *  from './useGetLectureList';
```
Comment (likegitman) [inline shared/api/common/src/hooks/lecture/index.ts]: 왜 export에 이것밖에 없죠?

File: shared/api/common/src/hooks/lecture/useDeleteRejectLecture.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    () => del(lectureUrl.lectureReject(id), {}),
+    {
+      onSuccess: (response) => {
+        console.log(response.data)
+      },
+      onError: (error) => {
+        console.log(error)
+      },
+    }
+  )
```
Comment (likegitman) [inline shared/api/common/src/hooks/lecture/useDeleteRejectLecture.ts]: 이제보니 tokenManager를 안 쓰는 것 같아서 지워주세요.

File: shared/api/common/src/hooks/lecture/usePostApplicationLecture.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    () => patch(lectureUrl.lectureApplication(id), {}),
+    {
+      onSuccess: (response) => {
+        console.log(response)
+      },
+      onError: (error) => {
+        console.log(error)
+      },
+    }
+  )
```
Comment (likegitman) [inline shared/api/common/src/hooks/lecture/usePostApplicationLecture.ts]: 여기서는 tokenManager가 필요 없는 것 같은데요?

File: shared/api/common/src/hooks/lecture/index.ts (hunk 1 above)
Comment (TaerangLee) [reply shared/api/common/src/hooks/lecture/index.ts]: [14364c5](https://github.com/GSM-MSG/Bitgouel-Frontend/pull/33/commits/14364c560ae1d9a05bdc0cddf260471a6fd2b8b3)
수정했습니다!

File: shared/api/common/src/hooks/lecture/useDeleteRejectLecture.ts (hunk 2 above)
Comment (TaerangLee) [reply shared/api/common/src/hooks/lecture/useDeleteRejectLecture.ts]: [7021ded](https://github.com/GSM-MSG/Bitgouel-Frontend/pull/33/commits/7021dedb5a58a76992ee14067f4055370911cf36)
[6931c4f](https://github.com/GSM-MSG/Bitgouel-Frontend/pull/33/commits/6931c4f63fe3e2b13cdf32a64489b744277ea975)

삭제하였습니다!

File: shared/api/common/src/hooks/lecture/usePostApplicationLecture.ts (hunk 3 above)
Comment (TaerangLee) [reply shared/api/common/src/hooks/lecture/usePostApplicationLecture.ts]: [cf08d05](https://github.com/GSM-MSG/Bitgouel-Frontend/pull/33/commits/cf08d052f92776a3ab16707efbdbdef924c0fd28)
수정하였습니다.

**Review by likegitman — APPROVED** [review]

**Review by googoo81 — APPROVED** [review]

**Pull request merged**